### DISCUSSION TITLE
[Question]: Documentation for deploying to kubernetes

### DISCUSSION THREAD

**Discussion by dcieslak19973** [danny-avila/LibreChat#3215] [Q&A] [answered]
### What is your question?

Admittedly I've only done a quick search, but I haven't been able to find docs on how to deploy LibreChat to k8s.  Does it exist?

### More Details

Documentation for deploying to k8s

### What is the main subject of your question?

Documentation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
None exists
